code/modules/antagonists/ashwalker/ashwalker.dm

```diff
@@ -28,6 +28,7 @@
 	. = ..()
 	RegisterSignal(owner.current, COMSIG_MOB_EXAMINATE, PROC_REF(on_examinate))
 	owner.teach_crafting_recipe(/datum/crafting_recipe/skeleton_key)
+	ADD_TRAIT(owner, TRAIT_SETTLER, ROLE_ASHWALKER)
```

Comment:
Also untested code alert, you're putting this trait on the mind of the mob, not the mob itself. 

Also also this should really be in `apply_innate_effects` / `remove_innate_effects`.

Reply:
Well now that you mention it, I see that it's their species that makes it so ashwalkers can't use the shuttle consoles. Shouldn't that be something determined either by their mind, or perhaps their brain? And the same with settler stuff.

Reply:
Or as you say, inside apply_innate_effects

Reply:
Could be a brain trait, it'd fit